front/lib/api/tables.ts

```diff
@@ -297,75 +316,50 @@ export async function rowsFromCsv(
     });
   }
 
+  const headerRes = await detectHeaders(
+    auth,
+    csv,
+    delimiter,
+    useAppForHeaderDetection
+  );
+
+  if (useAppForHeaderDetection) {
+    // Enable static header detection for debugging
+    const headerResStatic = await detectHeaders(auth, csv, delimiter, false);
+    logger.info(
+      { headerRes, headerResStatic, useAppForHeaderDetection },
+      "Header detection result"
+    );
+  } else {
+    logger.info(
```

Comment:
This log leaks user data and is kinda big. Not great

Reply:
right, removing logs here